code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -703,9 +714,4 @@ can cause issues with ammo types getting mixed up during the burst.
 
 	fire_delay = 6
 	accuracy_mult = 1.4
-	accuracy_mult_unwielded = 0.7
-	scatter = 15
-	scatter_unwielded = 40
-	recoil = 2
-	recoil_unwielded = 4
```

Comment:
Are these changes intended?

Reply:
Since the MBX share the same parent as the lever action, with the stats above heavily derived from it, I think it's not needed to copy the same thing and put it on the child weapon (the MBX).